packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -41,6 +42,10 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, Semver {
     /// @notice The trusted L2OutputOracle contract.
     L2OutputOracle public immutable L2_OUTPUT_ORACLE;
 
+    /// @notice The block hash oracle, used for loading block hashes further back
+    ///         than the `BLOCKHASH` opcode allows as well as their estimated timestamps.
+    BlockOracle public immutable BLOCK_ORACLE;
```

Comment:
Note that this doesn't follow our updated ABI guidelines where we don't leak the impl details into the ABI

Reply:
Will update all dispute immutables to be compliant in a separate PR - there are others as well.